src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensRangeTests.cs

```diff
@@ -225,7 +225,7 @@ void M()
                        0,     4,     1,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.LocalName],             0, // 'x'
                        0,     2,     1,    SemanticTokensHelpers.TokenTypeToIndex[LSP.SemanticTokenTypes.Operator],               0, // '='
                        0,     2,     5,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.VerbatimStringLiteral], 0, // '@"one'
-                       1,     0,     6,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.VerbatimStringLiteral], 0, // 'two'
+                       1,     0,     4,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.VerbatimStringLiteral], 0, // 'two '
                        0,     4,     2,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.StringEscapeCharacter], 0, // '""'
```

Comment:
the above was bad.  teh user code had `two ""` in it.  Previously, we generated two semantic tokens.  One for the full `two ""` and for the `""`.  This makes no sense in semantic tokens as things cannot overlap like this.  I'm not sure how this didn't break the client entirely as the semantic tokens are just completely broken here.

Reply:
note: 

![image](https://user-images.githubusercontent.com/4564579/207123096-d6e8b4ae-50d0-409e-8969-ec9a1ac0b52f.png)

this comment was a lie. :)

Reply:
The VS client unofficially supports overlapping tokens iirc, although I agree it's best to avoid.